Multisig/UI/Settings/Passcode/ChangePasscodeFlow.swift

```diff
@@ -32,10 +32,12 @@ class ChangePasscodeFlow: UIFlow {
             case .close:
                 stop(success: false)
             case .success(let passcode):
-                guard let passcode = passcode else {
-                    App.shared.snackbar.show(error: GSError.FailedToChangePasscode(reason: "Passcode required"))
-                    stop(success: false)
-                    return
+                if AppConfiguration.FeatureToggles.securityCenter {
```

Comment:
why is passcode not required when securityCenter toggle is off? Some comments in code would be helpful